no-computer.md

```diff
@@ -1,69 +1,175 @@
 # No computer project
 
+Running Routes for the Hash House Harriers
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
-
+This section describes a potential language, the group it serves, and some
+attributes of how the potential language is used.
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
 
+Periodically the organization know as the Hash House Harriers likes to lead fun
+running events. On such an event the organization would like to
+   1. Have one or two of its members come up with a set of activities for such
+      a run, along with a theme. That group, the _hares_ are responsible for
+      making a route and communicating it to the rest of the group, the
+      _hounds_.
+   2. Have the hounds understand and follow the route, doing fun activities
+      along the way.
+
+Preferably the route will also have constructs which keep the hounds moving as
+a group, regardless of athletic ability.
 
-### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
+### How is this need met?
 
+The Hash House Harriers solve this problem which I am going to presumptuously
+call the Hashing Language (HL). It is a formal system of communication from the
```

Comment:
Hash House Harriers is an actual group. The notation is real, but the name I use (HL) is unique to this write-up